core/dynamo.c

```diff
@@ -2571,7 +2571,7 @@ dynamo_thread_exit_common(dcontext_t *dcontext, thread_id_t id,
 #endif
 
 #ifdef DEBUG
-    if (dcontext->logfile != INVALID_FILE) {
+    if (dcontext->logfile != INVALID_FILE && dcontext->logfile != STDERR) {
```

Comment:
Could we add a simple test for this?  I assume it only matters on detach so a test where the app writes after the detach, which we probably already have but it's just not run with logging let alone -log_to_stderr.  Maybe it should just be in the long suite and not the CI suite.

Reply:
Can you advise on how to add a long suite test? I've looked through the cmake config and discovered some parts of the process, but I cannot for the life of me figure out where the test goes or how to mark it as LONG.

Reply:
https://github.com/DynamoRIO/dynamorio/blob/master/suite/tests/CMakeLists.txt#L97

Reply:
Going through old PR's: let's just get this fix in.